lib/gcp/rules.js

```diff
@@ -20,24 +23,31 @@ function _handleErrorResponse(response) {
 /**
  * Gets the latest ruleset name on the project.
  * @param {String} projectId Project from which you want to get the ruleset.
+ * @param {String} service Service for the ruleset (ex: cloud.firestore or firebase.storage).
  * @returns {String} Name of the latest ruleset.
  */
-function getLatestRulesetName(projectId) {
+function getLatestRulesetName(projectId, service) {
   return api
-    .request("GET", "/" + API_VERSION + "/projects/" + projectId + "/rulesets", {
+    .request("GET", "/" + API_VERSION + "/projects/" + projectId + "/releases", {
       auth: true,
       origin: api.rulesOrigin,
-      data: {
-        pageSize: 1,
-      },
     })
     .then(function(response) {
       if (response.status == 200) {
-        if (response.body.rulesets && response.body.rulesets.length > 0) {
-          return response.body.rulesets[0].name;
-        }
+        if (response.body.releases && response.body.releases.length > 0) {
+          var releases = _.orderBy(response.body.releases, ["updateTime"], ["desc"]);
 
-        return undefined;
+          var prefix = "projects/" + projectId + "/releases/" + service;
+          var release = _.find(releases, function(r) {
+            return r.name.indexOf(prefix) == 0;
+          });
+
+          if (!release) {
+            throw new FirebaseError("Can't find rules release for service " + service);
```

Comment:
You don't actually want to throw an error here. If the user doesn't have Firestore rules already, you would want to return null here, so that https://github.com/firebase/firebase-tools/blob/master/lib/init/features/firestore.js#L76 kicks in and the user gets default rules.